Update README.md with TorchTune reference (#2873)
pytorch/executorch: examples/models/llama2/README.md

    python -m examples.models.llama2.tokenizer.tokenizer -t tokenizer.model -o tokenizer.bin
    ```

## (Optional) Finetuning

If you want to finetune your model based on a specific dataset, PyTorch provides [TorchTune](https://github.com/pytorch/torchtune) - a native-Pytorch library for easily authoring, fine-tuning and experimenting with LLMs.

> Once you have [TorchTune installed](https://github.com/pytorch/torchtune?tab=readme-ov-file#get-started) you can finetune Llama2 7B model using LoRA on a single GPU, using the following command:

Reviewer: ```suggestion
Once you have [TorchTune installed](https://github.com/pytorch/torchtune?tab=readme-ov-file#get-started) you can finetune Llama2 7B model using LoRA on a single GPU, using the following command. This will produce a checkpoint where the LoRA weights are merged with the base model and so the output checkpoint will be in the same format as the original Llama2 model.
```

tune run lora_finetune_single_device \
--config llama2/7B_lora_single_device \
checkpointer.checkpoint_dir=<path_to_checkpoint_folder>  \

> tokenizer.path=<path_to_checkpoint_folder>/tokenizer.model

Reviewer: nit: Should wrap this in a code block (sorry suggestions doesnt allow me to do that)

> 

Reviewer: ```suggestion
To run full finetuning with Llama2 7B on a single device, you can use the following command.

tune run full_finetune_single_device \
--config llama2/7B_full_single_device \
checkpointer.checkpoint_dir=<path_to_checkpoint_folder> \
tokenizer.path=<path_to_checkpoint_folder>/tokenizer.model
```

    python -m examples.models.llama2.tokenizer.tokenizer -t tokenizer.model -o tokenizer.bin
    ```

> If you want to fine tune your model based on a specific dataset, PyTorch team provides a native-Pytorch library, [TorchTune](https://github.com/pytorch/torchtune), for easily authoring, fine-tuning and experimenting with LLMs.
> 

Reviewer: ```suggestion

## (Optional) Finetuning

If you want to finetune your model based on a specific dataset, PyTorch team provides [TorchTune](https://github.com/pytorch/torchtune) - a native-Pytorch library for easily authoring, fine-tuning and experimenting with LLMs.
```

Reviewer: Please feel free to disregard this suggestion.

Reviewer 2: @kartikayk can you provide an example one-line-ish command to generate a new fine-tuned state_dict pth file?

Reviewer: Oh sure, let me add this in the suggestion

Reviewer: Done

Author: Thanks @kartikayk ! Will update.
